src/core/error_handling.c

```diff
@@ -146,11 +139,19 @@ wrap_exception(bool attach_python_error)
   return jserror;
 }
 
+EM_JS_NUM(errcode, log_python_error, (JsRef jserror), {
+  let msg = Module.hiwire.get_value(jserror).message;
+  console.warn("Python exception:\n" + msg + "\n");
```

Comment:
In the future we should find a way to support custom error callbacks as we do in pyodide.js https://github.com/iodide-project/pyodide/blob/a31f0c4e39599ead2523c72b6ed19d20e68780ca/src/pyodide.js#L521

Reply:
What about using `sys.excepthook`?

Reply:
You mean user is expected to monkeypatch it?  Could be as long as it's documented.